src/pyvesync/vesync.py

```diff
@@ -524,15 +525,15 @@ def __init__(self, details, manager):
         super(VeSyncSwitch15A, self).__init__(details, manager)
 
     def get_body(self, type_):
-        if type_ == 'details':
```

Comment:
Is there a typo somewhere or are you just switching the type_ value passed from 'details' to detail'?

Reply:
I was switching it for consistency elsewhere in the same file. I can back that out if you'd like.

Reply:
No that's a great catch.